ui/src/components/ActiveAlertsFilters.js

```diff
@@ -41,65 +17,42 @@ const ActiveAlertsFilter = (props) => {
   let items = [
     {
       label: 'All',
-      value: 'all',
       onClick: () => {
         query.delete('severity');
         history.push(`${match.url}?${query.toString()}`);
       },
-      iconCode: '',
-      'data-cy': 'severity_all',
+      selected: !selectedFilter,
     },
     {
-      label: 'Critical',
-      value: STATUS_CRITICAL,
+      label: 'Health',
       onClick: () => {
-        query.set('severity', STATUS_CRITICAL);
+        query.set('severity', STATUS_HEALTH);
         history.push(`${match.url}?${query.toString()}`);
       },
-      iconCode: 'fas fa-times-circle',
-      'data-cy': 'severity_critical',
+      selected: selectedFilter === STATUS_HEALTH,
     },
     {
       label: 'Warning',
-      value: STATUS_WARNING,
       onClick: () => {
         query.set('severity', STATUS_WARNING);
         history.push(`${match.url}?${query.toString()}`);
       },
-      iconCode: 'fas fa-exclamation-triangle',
-      'data-cy': 'severity_warning',
+      selected: selectedFilter === STATUS_WARNING,
+    },
+    {
+      label: 'Critical',
+      onClick: () => {
+        query.set('severity', STATUS_CRITICAL);
+        history.push(`${match.url}?${query.toString()}`);
+      },
+      selected: selectedFilter === STATUS_CRITICAL,
     },
   ];
 
-  const dropDownLabel = (() => {
-    if (!selectedFilter) return items[0].label;
-
-    const item = items.find((item) => item.value === selectedFilter);
-    return (
-      <span>
-        {item.value !== 'all' && (
-          <FilterIcon status={item.value} className={item.iconCode} />
-        )}
-        {item.label}
-      </span>
-    );
-  })();
-
-  if (selectedFilter) {
-    items = items.filter((item) => item.value !== selectedFilter);
-  } else {
-    items.splice(0, 1);
-  }
-
   return (
-    <ActiveAlertsFilterWrapper>
-      <Dropdown
-        items={items}
-        text={dropDownLabel}
-        size="small"
-        data-cy="alert_filter"
-      />
-    </ActiveAlertsFilterWrapper>
+    <div data-cy="alert_filter">
+      <Healthselector items={items} />
+    </div>
```

Comment:
In my opinion, we should add the `data-cy` directly to `<Healthselector/>` so we don't need to adjust the code in the test.

Reply:
@ChengYanJin That's what I did initially but seems `<Healthselector/>` is directly exported from core-ui the custom HTML attributes such as `data-cy` are not forwarded, that's why the tests were failing. Do you have an other way around that ? :)

Reply:
@alexis-ld  one thing we can do is to add the spread syntax to expand the propery of the component in core-ui.
```js
function Healthselector({ size, items, ...rest }: Props) {
  return (
    <HealthselectorContainer className="sc-healthselector" {...rest}>
}
```